ppr-ui/src/components/mhrRegistration/ReviewConfirm/YourHomeReview.vue

```diff
@@ -179,6 +179,16 @@ export default defineComponent({
       }),
       engineerDisplayDate: computed((): string => {
         return yyyyMmDdToPacificDate(getMhrRegistrationHomeDescription.value?.engineerReportDate, true)
+      }),
+      manufactureYear: computed((): string => {
```

Comment:
Not sure if we need a computed prop for this. We can simplify and use at the template level. E.g.
```
<p v-if="getMhrRegistrationHomeDescription.baseInformation.year">
...Check for circa here...
</p>
<p v-else>
(Not Entered)
</p>
```

Reply:
Depends on what the check is,
I'm in the camp of keeping logic out of the template but if it can be done within a basic ternary than all good. Your call but less logic in the template the better 👍

Reply:
oops pushed before seeing your comment again; anyways, let me know if you would like to revert the changes.